lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -393,7 +393,13 @@ def insert_at_position(position)
         end
 
         def update_positions
-          old_position = send("#{position_column}_was") || bottom_position_in_list + 1
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
I can do either way, but @brendon made a (valid) suggestion that version comparison makes it easier to remove old code path in the future.

That's what's currently done in:
https://github.com/swanandp/acts_as_list/blob/f9b40e1cbdefe5244b787c66da74148e351af26c/lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb#L8
(which could be replaced w/ eventual `activerecord_version_at_least?('5')`).